sdk/tests/conformance2/extensions/webgl-multi-draw-instanced-base-vertex-base-instance.html

```diff
@@ -948,24 +928,24 @@
     function drawElementsInstancedBaseVertexBaseInstanceWEBGL(gl) {
       const ext = gl.getExtension('WEBGL_draw_instanced_base_vertex_base_instance');
       if (!ext) {
-        return true;
+        throw 'Should not have run this test without WEBGL_draw_instanced_base_vertex_base_instance';
       }
 
-      ext.drawElementsInstancedBaseVertexBaseInstanceWEBGL(gl.TRIANGLES, 6, gl.UNSIGNED_BYTE, 0, 1, 0, 1);
+      ext.drawElementsInstancedBaseVertexBaseInstanceWEBGL(gl.TRIANGLES, 6, gl.UNSIGNED_BYTE, 0, 1, 0, 0);
```

Comment:
Note: it looked to me like the original code here was wrong - specifying 1 rather than 0 for the base instance. However, the tests pass with either version of this code.

Reply:
lgtm your way